From a4b55a7943160cf07928e551b766432d4df93a8a Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <daniel@octaforge.org>
Date: Wed, 16 Dec 2020 07:36:38 +0100
Subject: [PATCH] explicitly check __powerpc64__ in floatn.h for ppcle builds

otherwise the build is spammed with warnings about _CALL_ELF not
being defined when building for ppcle, which is harmless but still
annoying
---
 sysdeps/powerpc/bits/floatn.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sysdeps/powerpc/bits/floatn.h b/sysdeps/powerpc/bits/floatn.h
index cb97184..13e473b 100644
--- a/sysdeps/powerpc/bits/floatn.h
+++ b/sysdeps/powerpc/bits/floatn.h
@@ -25,7 +25,8 @@
 /* Defined to 1 if the current compiler invocation provides a
    floating-point type with the IEEE 754 binary128 format, and this glibc
    includes corresponding *f128 interfaces for it.  */
-#if defined _ARCH_PWR8 && defined __LITTLE_ENDIAN__ && (_CALL_ELF == 2) \
+#if defined _ARCH_PWR8 && defined __LITTLE_ENDIAN__ \
+    && defined __powerpc64__ && (_CALL_ELF == 2) \
     && defined __FLOAT128__ && !defined __NO_LONG_DOUBLE_MATH
 # define __HAVE_FLOAT128 1
 #else
-- 
2.29.2

